renovate-config.json5

```diff
@@ -58,6 +58,11 @@
       ],
       dependencyDashboardApproval: true,
     },
+    {
+      // disable volta (i.e. pinned Node and npm version) updates
+      matchDepTypes: ['volta'],
```

Comment:
**issue:** Doesn't `matchDepTypes` only encompass npm dependency types like `peerDependencies` and `optionalDependencies`?

Unfortunately, it looks like the `supportPolicy` field we were using to set node versions has recently been [removed](https://github.com/renovatebot/renovate/pull/11176) (and perhaps never worked seeing as [we don't use Travis](https://github.com/renovatebot/renovate/issues/11144)...?) According to the [docs](https://docs.renovatebot.com/node/), as far as I can see the only way to disable these renovations are by explicitly pinning the node version in the `engines` field in a repository's `package.json`, though hopefully I'm missing a way to disable it in the config too!

Reply:
according to [this](https://docs.renovatebot.com/modules/manager/npm/#additional-information), "`depTypes`" includes `volta`. now, there's no actual option called `depTypes`, so i assume this refers to `matchDepTypes`.

it's kind of hilarious that `supportPolicy` was only for Travis. i could remove it in this PR?

Reply:
ah yep I completely missed that part of the docs, that's a big relief!! (wish they'd linked to that from either the node page I linked or from the `matchDepTypes` definition...)

> it's kind of hilarious that supportPolicy was only for Travis. i could remove it in this PR?

probably makes sense, might avoid some confusion in the future